### PR TITLE
navbar and list of post

### PR DESCRIPTION
**Pull request by stevenfung30**

### CODE REVIEW

**Review by Ben0189 — CHANGES_REQUESTED** [review]
dont commit node module

File: package-lock.json
Comment (Ben0189) [inline package-lock.json:1]: Do we need this ? can you remove this file?

File: package.json
Comment (Ben0189) [inline package.json:1]: Do we need this ? can you remove this file?

File: tailwind.config.js
Comment (Ben0189) [inline tailwind.config.js:1]: Do we need this ? can you remove this file?

File: postcss.config.js
Comment (Ben0189) [inline postcss.config.js:1]: Do we need this ? can you remove this file?

File: career-link-webui/src/app/listofpost.tsx
Comment (Ben0189) [inline career-link-webui/src/app/listofpost.tsx:1]: rename this file from listofpost to list-of-post

**Review by Ben0189 — CHANGES_REQUESTED** [review]

**Comment by Ben0189** [comment]
good work , can you also post the UI in this PR?

**Review by Ben0189 — APPROVED** [review]

**Comment by stevenfung30** [comment]
image

**Pull request merged**